content/blog/pragmatism-over-perfection.md

```diff
@@ -0,0 +1,120 @@
+
+However there were some constraints:
+
+1. **Missing Security Features:** Most open-source tools lacked the security and access controls we needed, at least in their free versions. And buying licenses didn’t make sense for a one-off task, this meant that I would need to somehow implement security and access control.
+
+2. **Time pressure:** The images were ready to be tagged. The stakeholders were waiting. And the Ops team was blocked on me to provide them with an interface. Setting up the infrastructure, securing it, and onboarding users would take time we did not have.
+
+These constraints made the "right" tools impractical for our needs.
+
+## A Hack that Worked
+
+This is when the shift happened. I stepped back and looked at what we already had, and an idea struck.
+
+We were already using Databricks internally. What if it could be turned into a tagging tool?
+
+I pitched the idea to my manager. It was hacky, no doubt, but it was great.
+It ticked all our boxes:
+
+1. **Access control was sorted**: Databricks had SSO and RBAC baked in.
+
+2. **Onboarding was simple**: The process was documented and quick.
+
+3. **The KYC images were already available** inside Databricks to people with the right access.
+
+4. **No new infrastructure was needed**: We already had it running, hence saving time, setup overhead, and additional infrastructure costs.
+
+5. **Time to go live was minimal**: I could get the Ops team started almost immediately, contributing to the time saved on the larger goal, model deployment.
+
+My manager signed off, and we presented the options to the stakeholders. They were happy that we could use an existing system to get the job done.
+So, we used Databricks as a tagging tool.
+
+It was in no way the "right" tool for the job, but it worked, and that made it the best one.
+And now it was time to implement this pragmatic solution.
+
+> What is databricks?
+>
+> Databricks is an all in one platform for analysts, and engineers to manipulate, process and use data, read more [here](https://www.databricks.com/data-intelligence?scid=7018Y000001f8FIQAY&utm_medium=paid+search&utm_source=google&utm_campaign=20782149301&utm_adgroup=152953302702&utm_content=microsite&utm_offer=data-intelligence&utm_ad=724408738477&utm_term=what%20is%20databricks&gad_source=1&gclid=Cj0KCQjw2ZfABhDBARIsAHFTxGwAa41AMcCUzaTbsL60svmAaD4LReAsmqlwm_SMoJYbKgzcDWwEoGAaAi4wEALw_wcB).
+
+## Getting it up and Running
+
+I onboarded the Ops team onto Databricks and assigned them the right roles, which was quick and easy since I already had admin access.
+
+Then, I created a simple notebook for them.
+Here is what the notebook did:
+
+1. Fetched an image path at random from the table we had where the manual tag field was Null.
```

Comment:
this is the first time table is mentioned, could you add more details around it so that it becomes clear what you are talking about? It feels like you expect the readers to know about the table already.